src/main/java/exception/BookNotFoundException.java

```diff
@@ -1,7 +1,12 @@
+package exception;
+
 public class BookNotFoundException extends RuntimeException {
 
     public BookNotFoundException(String message, Throwable cause) {
         super(message, cause);
     }
+
+    public  BookNotFoundException(){
+    }
```

Comment:
Added an empty constructor because that's how you were using it inside `BookController`: _line 47_